var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -88,9 +84,6 @@ class Gcc(AutotoolsPackage):
     def configure_args(self):
         spec = self.spec
         prefix = self.spec.prefix
-        # libjava/configure needs a minor fix to install into spack paths.
-        filter_file(r"'@.*@'", "'@[[:alnum:]]*@'", 'libjava/configure',
-                    string=True)
```

Comment:
I'm pretty sure this was left over from back when Spack's installation directories contained characters like `@`. It shouldn't be needed anymore.